tests/extension/oneapi_device_global/device_global_common.h

```diff
@@ -117,6 +128,17 @@ struct value_helper<T[N]> {
       value[i] = new_val;
     }
   }
+  /**
+   * @brief The function changes all values of the array from the first parameter to
+   * values of the array from the second parameter
+   * @param value The reference to the array that needs to be change
```

Comment:
All PRs with functional tests have common code that using in all tests, so I cherry-picked commit with common code in every PR. I plan to pull changes from SYCL-2020 when one of the PR will be merged. Until then, I left the link to related changes to avoid the déjà vu effect :)
Anyway applied fix to this branch and other functional tests branches too.